core/lib/spree/permitted_attributes.rb

```diff
@@ -62,6 +64,11 @@ module PermittedAttributes
 
     @@line_item_attributes = [:id, :variant_id, :quantity]
 
+    @@menu_attributes = [:name, :locale, :location, :store_id]
```

Comment:
```suggestion
    @@menu_attributes = [:name, :locale, :location]
```

we should not allow manipulating the store_id